README.md

```diff
@@ -23,6 +23,20 @@ rosinstall --catkin . uwreact_robot/uwreact_robot.rosinstall
 rosdep install --from-paths . --ignore-src -r -y
 ```
 
+In order to build anything zed related, you'll also need to install CUDA 10 and the ZED SDK
+
+### CUDA 10
+
+[Download CUDA 10](https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&target_distro=Ubuntu&target_version=1804&target_type=deblocal) and follow install instructions according to which installation method you prefer
```

Comment:
I would still prefer to see this without the query string, but not a big deal. Approved either way.

Reply:
Sure, i can remove it, just thought I'd leave it for less confusion but its pretty straight forward